gpustack/routes/model_instances.py

```diff
@@ -107,7 +107,7 @@ async def proxy_stream():
             if resp.status != 200:
                 raise HTTPException(
                     status_code=resp.status,
-                    detail="Error fetching serving logs",
+                    detail=f"Error fetching serving logs: {resp.reason}",
```

Comment:
I think the problem is client.get() raises exceptions but we didn't catch them. Use the `try` clause to handle exceptions properly.